admin-dev/themes/new-theme/js/pages/specific-price/specific-price-map.ts

```diff
@@ -24,9 +24,14 @@
  */
 
 export default {
+  currencyId: '#specific_price_currency_id',
   customerSearchContainer: '#specific_price_customer',
   priceInput: '#specific_price_fixed_price',
+  priceSymbol: '.js-fixed_price-row .input-group.money-type .input-group-append .input-group-text, '
```

Comment:
I guess you need to target both prepend and append classes because the MoneyType form field changes its display based on the language? Maybe it could be simplified by updating the form theme and adding a single class that would be common to both use cases? (just a suggestion though)

Reply:
I think it can be done in the future. Isn't it ?

Reply:
Yeah absolutely